skbio/stats/composition.py

```diff
@@ -1105,6 +1211,64 @@ def _gram_schmidt_basis(n):
     return basis.T
 
 
+def _sbp_basis(sbp):
```

Comment:
You sure you want this to be a private method?  It seems like it would be more appropriate to have this be a publicly available method.